### PR TITLE
refactor: main

### PR DESCRIPTION
**Pull request by NeonGamerBot-QK**
- removed unused tests, and webvitals since https://github.com/hackclub/counterspell/commit/dbfd6c08ecd23d5ca78751c1e64f2e070c3ed35b exists.
- fix manifest.json to use correct name & desc

*note: zeon is my bot and only linted the files with prettier*

### CODE REVIEW

File: src/reportWebVitals.js
Comment (samdev-7) [inline src/reportWebVitals.js:1]: deployment is failing due to this file missing, it seems

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by NeonGamerBot-QK** [comment]
Fixed @samdev-7

**Comment by NeonGamerBot-QK** [comment]
@samdev-7 could u find the err? (i cant see vercels thing)

**Comment by samdev-7** [comment]
Hopefully this helps!

**Comment by NeonGamerBot-QK** [comment]
@samdev-7 fixed.

**Comment by samdev-7** [comment]
Thanks for the PR but I won't be merging this. I'll be formatting these files as I go, and the format seems to conflict with what I have set up (prettier default).

**Pull request closed without merging**